activiti-cloud-query-service/activiti-cloud-starter-query/src/main/resources/config/query/liquibase/changelog/06-alter.oracle.schema.7.1.0-m10.sql

```diff
@@ -0,0 +1,6 @@
+alter table task
+    add column completed_by varchar(255);
```

Comment:
Syntax error for Oracle. It should be `add completed_by` instead of `add column completed_by`.

Reply:
Good catch.